sensor.py

```diff
@@ -26,6 +26,27 @@
 
 TIME_ZONE = "Z" if str(dt_util.DEFAULT_TIME_ZONE) == "UTC" else str(dt_util.DEFAULT_TIME_ZONE)
 
+def get_from_to_datetime_month(months=1):
+    """Get a period
+    This will return two dates in ISO 8601:2004 format
+    The first date will be at 00:00 in the first of this month, and the second
+    date will be at 00:00 in the first day of next month.
+
+    Both dates include the time zone name, or `Z` in case of UTC.
+    Including these will allow the API to handle DST correctly. 
+
+    When asking for measurements, the `from` datetime is inclusive
+    and the `to` datetime is exclusive. 
+    """
+    #to_dt = datetime.datetime.now()
+    from_dt = datetime.datetime.now().replace(day=1, hour=0, minute=0, second=0, microsecond=0)
+    to_dt = from_dt + datetime.timedelta(days=31)
```

Comment:
This will probably not work as expected, as it will spill over in the next month if it is less than 31 days.

Reply:
I just re-read the pull request comment about this, can you please add a comment that it's a non-issue to use 31 days if you're keeping it?

Reply:
Added comment, and fixed  so its properly is just the month we return from 1st to 1st.